keps/sig-network/3698-multi-network-requirements/README.md

```diff
@@ -0,0 +1,352 @@
+Primary PodNetwork.
+
+<p align="center">
+  <img src="mn-story-8.png?raw=true" alt="multi-network story 8 network interface hot-plug"/>
+</p>
+
+#### Story #9
+As an infrastructure administrator I wish to support IPv6 SLAAC in the
+Kubernetes Pods. SLAAC's nature is to dynamically assign IP addresses and change
+them over time. Those should be reflected in Pod status. This would not be
+applicable for the Primary PodNetwork.
+
+### Requirements
+
+Below requirements are divided into phases. Each phase will produce a separate
+KEP with detailed design for specified set of requirements. Each of this KEP
+will have to take all of the requirements, from each phase, into consideration
+when created.
+
+#### Phase I (base API and reference in Pod)
+1. This effort shall not change the behavior of Today existing clusters
+2. We need an API object representing network, which a Pod, a Service or any
+other feature that wishes to support Multi-Network capability could reference
+   * We will introduce an PodNetwork API object that represents the existing 
+     infrastructure’s networking
+3. PodNetwork is decoupled from the infrastructure’s network
+definition/description created by the cluster operator. It functions as a handle
+4. PodNetwork shall not define any implementation specific parameters in its
+specification
+5. PodNetwork shall provide cluster generic options only
+6. PodNetwork can reference to implementation-specific parameters
+7. PodNetwork is the object Application developer will reference in their workloads
+8. Cluster Default PodNetwork is the PodNetwork the cluster has been created with
+9. Cluster Default PodNetwork cannot be changed or removed 
+10. Pods can reference one or more PodNetworks when trying to attach to them
+    * This list is explicit and all the PodNetworks that has to be available in
+      the Pod has to be listed
+11. Workloads access to PodNetwork is controlled via RBAC configuration
+12. When Pods does not specify any reference to PodNetwork it connects to Cluster
+Default PodNetwork (network the cluster has been created with)
+13. Pod shall be able to provide additional configuration on how it attaches to
+a PodNetwork
+14. Every Pod connected to specific PodNetwork must have connectivity within
+that network across the Cluster
+    * A Pod connected to specific PodNetwork may or may not have cross
+      connectivity between different PodNetworks
+15. Pods attached to PodNetworks are connected to each other in a manner 
+defined by the PodNetwork implementer
+16. Basic PodNetwork Interface information for each attachment will be exposed
+to runtime Pod (via e.g. environment variables, downward API etc.)
+
+#### Phase II (scheduler, kubelet and API probing for PodNetwork)
+17. PodNetworks can be selectively (per Node) available in the Cluster. This
+does NOT apply to the Cluster Default PodNetwork
+18. Kubelet network-based probing can be optionally enabled for the non- Cluster
```

Comment:
Does this refer to Kubelet-based healthchecks? Would help to link to the feature more explicitly.

Reply:
This refers to all the possible network-based communication kubelet has with the Pod via Today's eth0. Are healthchecks the only thing?